db/schema.rb

```diff
@@ -10,15 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180417220239) do
+ActiveRecord::Schema.define(version: 20180420012423) do
 
-  create_table "categories", force: :cascade do |t|
+  create_table "base_categories", force: :cascade do |t|
     t.string "name", null: false
-    t.string "color"
+    t.string "color", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "categories", force: :cascade do |t|
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "description"
-    t.index ["name"], name: "index_categories_on_name", unique: true
+    t.string "locale", default: "pt-BR"
```

Comment:
Note: now that we're changing 'pt-BR' to 'pt', we need to update all 'pt-BR' references in this PR.